skyplane/obj_store/gcs_interface.py

```diff
@@ -58,6 +58,7 @@ def map_region_to_zone(region) -> str:
                     f"No access to the Google Cloud Storage bucket '{self.bucket_name}', assuming bucket is in the 'us-central1-a' zone"
                 )
                 return "us-central1-a"
+            raise e
```

Comment:
`raise e from None`?

Reply:
Updated to simply `raise` (see https://github.com/skyplane-project/skyplane/pull/661). I tested this out, and I'd like to preserve exception chaining for cloud SDKs but have disabled it with the `raise e from None` for first-party exceptions. Lmk your thoughts on this strategy.

Reply:
I reviewed that PR. The strategy makes sense to preserve cloud SDK exceptions :+1: